app/views/export/generic.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    .divider:nth-child(odd) {
```

Comment:
This is mainly to show unique sets of data within a card.  It's not the prettiest thing, but it easier to read than without it